debian/syslog-ng-core.syslog-ng.upstart

```diff
@@ -0,0 +1,32 @@
+# syslog-ng - system logging daemon
```

Comment:
This cannot be merged as the lintian will alert:

https://lintian.debian.org/tags/package-installs-deprecated-upstart-configuration.html

Reply:
we will need to maintain this upstream then. hopefully old ubuntu versions will fade away then we can drop it too.